WebApplication/Backend/Repositorys/BuildingRepository.cs

```diff
@@ -3,11 +3,11 @@
 using System.Collections.Generic;
 using HealthClinicBackend.Backend.Model.Hospital;
 using HealthClinicBackend.Backend.Repository.DatabaseSql;
-using WebApplication.Backend.Repositorys.Interfaces;
+using HealthClinicBackend.Backend.Repository.Generic;
 
 namespace WebApplication.Backend.Repositorys
 {
-    public class BuildingRepository : IBuildingRepository
+    public class BuildingRepository : IGenericRepository<Building>
```

Comment:
This is completely unnecessary, the repositories in web project should be completely deleted - you just need to use the ones in Backend project (ideally through controllers and services - MVC)